src/components/RollupGrantsForm.js

```diff
@@ -366,131 +511,158 @@ const RollupGrantsForm = props => {
           onChange={handleInputChange}
           maxLength="15"
         />
-      </Label>
-      <Label>
+      </StyledLabel>
+      <StyledLabel>
         <span>GitHub Username</span>
         <div>
           <small>Ex: @github_username</small>
         </div>
         <Input
           type="text"
           name="github"
-          value={formState.github}
+          value={formState.github.value}
           onChange={handleInputChange}
           maxLength="40"
         />
-      </Label>
-      <Label>
+      </StyledLabel>
+      <StyledLabel>
         <span>LinkedIn Profile URL</span>
         <div>
           <small>Ex: https://www.linkedin.com/in/profilename</small>
         </div>
         <Input
           type="text"
           name="linkedin"
-          value={formState.linkedin}
+          value={formState.linkedin.value}
           onChange={handleInputChange}
+          onBlur={handleTouched}
           maxLength="150"
         />
-      </Label>
-      <Label>
+        <ErrorDiv>
+          {formState.linkedin.isTouched && !formState.linkedin.isValid && (
+            <ErrorMessage>Ensure LinkedIn profile is valid URL</ErrorMessage>
+          )}
+        </ErrorDiv>
+      </StyledLabel>
+      <StyledLabel>
         <span>
           What questions do you have about anything related to rollups?
         </span>
         <TextArea
           name="questions"
-          value={formState.questions}
+          value={formState.questions.value}
           onChange={handleInputChange}
         />
-      </Label>
-      <Label>
-        <span>Telegram Username{/* or Alternative Contact Info */}</span>
+      </StyledLabel>
+      <StyledLabel>
+        <span>Telegram Username</span>
         <Input
           type="text"
           name="contactAlternative"
-          value={formState.contactAlternative}
+          value={formState.contactAlternative.value}
           onChange={handleInputChange}
           maxLength="150"
         />
-      </Label>
-      <Label>
+      </StyledLabel>
+      <StyledLabel>
         <span>Anything else you'd like to share? (optional)</span>
         <TextArea
           name="other"
-          value={formState.other}
+          value={formState.other.value}
           onChange={handleInputChange}
         />
-      </Label>
+      </StyledLabel>
       <Checkbox>
         <CheckboxInput
           type="checkbox"
           name="newsletter"
-          value={formState.newsletter}
+          value={formState.newsletter.value}
           onChange={handleCheckBoxChange}
         />
         Subscribe to the ESP Newsletter? You'll hear from us every few weeks,
         and we'll only ever contact you with ESP news.
       </Checkbox>
       <div>
-        <Button disabled={isButtonDisabled} type="submit">
+        <StyledButton disabled={isButtonDisabled} onClick={handleSubmit}>
           {buttonText}
-        </Button>
+        </StyledButton>
+        {isButtonDisabled && (
```

Comment:
```suggestion
        {!isFormValid() && (
```

Reply:
This resulted in the "Fill out all required fields before submitting" text displaying while the form was being submitted.